metrics/metrics.go

```diff
@@ -0,0 +1,66 @@
+package metrics
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+)
+
+var (
+	MOutgoingMsgs = stats.Int64("pubsub/outgoing_msg_size", "Outgoing message size", "By")
+	MIncomingMsgs = stats.Int64("pubsub/incoming_msg_size", "Incoming message size", "By")
```

Comment:
it's not a size!

Reply:
whooooooooops

Reply:
oh, actually the measure *is* a size, but i can aggregate it by count as well

Reply:
I think you can do this with an extra view with view.Count, on the same measure. Not 100% sure.

Reply:
We definitely want count, we don't care about sizing as much but we do care about _number of messages_.